Emby.Server.Implementations/Data/SqliteExtensions.cs

```diff
@@ -1,11 +1,12 @@
-#nullable disable
+#nullable enable
```

Comment:
```suggestio
```

Reply:
?

Reply:
Nullable is enabled by default, you can remove the first line